### PR TITLE
fixed-joincommunity button 

### LINKED ISSUE #69: [FEAT] Make join community button on website more dynamic and fix padding.
### What feature would you like to see?

image
The Join Button on website does not have proper padding and it doesnt have any effect when user hovers over it, adding basic hover effect in CSS might do the work.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by KaKashi1210**
# Pull Request

## Description

Added hover effect on join community button for visual cue.

## Related Issue

Closes #69

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
LGTM. Thanks! 🚀

**Comment by KaKashi1210** [comment]
@Panquesito7 Thanks for taking out time to review the commit <3.

File: public/index.css [hunk 1]
```diff
@@ -160,6 +160,12 @@ color: black;
   border: 1px solid black;
   font-size: 1.3rem;
   margin-top: 20px;
+  transition: ease-in-out 500ms;
+}
+.button:hover{
```
Comment (naomi-lgbt) [inline public/index.css:165]:
```suggestion
}

.button:hover{
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Nitpicking - but let's keep a blank line between style rules for readability

**Comment by KaKashi1210** [comment]
@naomi-lgbt done

**Comment by KaKashi1210** [comment]
do you want some other changes?

**Review by naomi-lgbt — APPROVED** [review]
banger pr, thanks for contributing!

**Comment by naomi-lgbt** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

You can also see all open issues [through our contributor tool](https://contribute.naomi.lgbt)

**Comment by KaKashi1210** [comment]
Commenting here for holopin badge!

**Pull request merged**